### ISSUE TITLE
Make pref_behaviour_notifications disable the alarm

### ISSUE THREAD

**Issue by alexzorin** [QuantumBadger/RedReader#204]
Don't really do Android but wouldn't it save some battery life/alarm invocations to just disable [the 30 min alarm](https://github.com/QuantumBadger/RedReader/blob/cbcc850bf862c9edcae055b3143e7343da7b8294/src/main/java/org/quantumbadger/redreader/common/Alarms.java#L40) if [notifications are disabled](https://github.com/QuantumBadger/RedReader/blob/cbcc850bf862c9edcae055b3143e7343da7b8294/src/main/java/org/quantumbadger/redreader/receivers/NewMessageChecker.java#L59-L60)?

At the moment my reading of it is 
* wake up
* realize I shouldn't have woken up
* go back to bed

**Comment by QuantumBadger** [comment]
You're right, although the amount of battery life used by this is basically none. The alarm is set using `setInexactRepeating()`, which means Android will schedule the alarm in sync with other alarms (so if you have five alarms, Android will wake up once to service all five). Also, only the CPU gets woken up (not the radio).

Even so, I'd happily accept a pull request that disabled the alarm.
